.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea/
```

Comment:
The gitignore changes shouldnt be here  - commit them on main and then merge main here

Reply:
so from now on I can merge a branch which contains the gitignore file without pull request?

Reply:
Yep :) You can merge main to any branch you want as much as you would like, you can merge other branches to other branches as much as youd like (unless you want a review there), open a pull request only when you are ready to merge to main

Reply:
or to production, on the real world